package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "gql_mini_project",
+  "version": "1.0.0",
+  "main": "app.js",
+  "repository": "https://github.com/ynaydenov-qb/gql_mini_project.git",
+  "author": "Yoan Naydenov <ynaydenov@quickbase.com>",
+  "license": "MIT",
+  "dependencies": {
+    "@graphql-tools/merge": "^9.0.6",
+    "@types/node": "^22.4.1",
```

Comment:
Not a major issue here but we tend to lock versions in order to avoid potential updates from breaking our code.

Reply:
I locked the versions in the newest package.json file, thanks